andela/healthchecks-thunknalla
# #146143817 Configure heroku deployment

#### What does this PR do?
Adds configuration files needed for the project to be deployed on heroku.

#### Description of Task to be completed?
- [x] Use the web interface to create heroku apps:
  - [x] [`hc-thunknallla-prd`](https://hc-thunknalla-prd.herokuapp.com)- production environment app.
  - [x] [`hc-thunknallla-dev`](https://hc-thunknalla-dev.herokuapp.com)- development app.
- [x] Configure git branches to be deployed for each of the apps:
  - [x] production is configured to deploy the [`develop`](https://github.com/andela/healthchecks-thunknalla/tree/develop) branch
  - [x] develop is configured to deploy any of the feature branches that need to be showcased

#### How should this be manually tested?
- Opening https://hc-thunknalla-prd.herokuapp.com on your favorite browser should load healthchecks-thunknalla at the state the develop branch is at.
- Opening https://hc-thunknalla-dev.herokuapp.com on your favorite browser should load healthchecks-thunknalla at the state the [branch](https://github.com/andela/healthchecks-thunknalla/tree/ch-setting_up_heroku_deployment-146143817) for this PR is at. At least until it's merged.

#### Any background context you want to provide?
- Since the [`develop`](https://github.com/andela/healthchecks-thunknalla/tree/develop) branch is the default branch for this repo, it represents the stable state of the project more than master would. Makes sense to deploy it to the production environment.
- While this PR has been initiated by @collinmutembei, the development work happened in pairing sessions to accelerate progress.

#### What are the relevant pivotal tracker stories?
The PT story for this work can be found [here](https://www.pivotaltracker.com/story/show/146143817)

#### Screenshots

prd environment
<img width="1677" alt="screen shot 2017-06-01 at 14 46 14" src="https://cloud.githubusercontent.com/assets/10899666/26678450/4202e86a-46d9-11e7-9781-20faedc5ab0b.png">
  
dev environment
<img width="1680" alt="screen shot 2017-06-01 at 14 46 31" src="https://cloud.githubusercontent.com/assets/10899666/26678468/51bb3fc8-46d9-11e7-806d-6acd45a46b72.png">
